fixd.js

```diff
@@ -1,14 +1,55 @@
-const fs = require('fs')
-const path = require('path')
-const fixd = require('./src/index')
+'use strict'
+const merge = require('lodash.merge')
```

Comment:
What about `clone` instead of `merge`? That way you can avoid your `Array.isArray` check and defer any polymorphism to lodash. Not a blocker, but an idea.